src/App.js

```diff
@@ -28,27 +28,34 @@ function App() {
   const addProject = ({ name = "" }) => {
     const newProject = { name, userId, datePracticed: Date.now() };
     console.log(newProject);
-    // axios
-    //   .post("#", newProject)
-    //   .then((response) => {
-    //     //TODO: Response here
-    //   })
-    //   .catch((error) => {
-    //     console.log("Error posting project", error);
-    //   });
+    axios
+      .post(`https://zlld6v728l.execute-api.eu-west-2.amazonaws.com/dev/projects`, newProject)
+      .then(({ data: { projects: resProject = [] } = {} }) => {
+        setProjects([...projects, ...resProject]);
+      })
+      .catch((error) => {
+        console.log("Error posting project", error);
+      });
   };
 
   const addSkill = (projectId, skillName) => {
     const newSkill = { name: skillName, projectId: projectId };
     console.log(newSkill);
-    // axios
-    //   .post("#", addSkill)
-    //   .then((response) => {
-    //     //TODO: Response here
-    //   })
-    //   .catch((error) => {
-    //     console.log("Error posting skill", error);
-    //   });
+    axios
+      .post(`https://zlld6v728l.execute-api.eu-west-2.amazonaws.com/dev/skills`, newSkill)
+      .then((response) => {
+        const updatedProjects = projects.map((project) => {
+          const { skills = [] } = project;
+          if (project.projectId === projectId) {
+            return { ...project, skills: [response.data.skill, ...skills] };
```

Comment:
This results in a new project not having the skillToDo property. This is currently only added when useEffect is called when the userId changes. How do we want to resolve this as this may also be an issue for if the user deletes the skill which happens to be the skillToDo today?

Reply:
I'm not sure what you mean by this, none of the existing data is being changed, the new skill is just being appended to the beginning of the skills array in the correct project - The skillToDo property remains after adding a skill